Modulo9-BrunoMonteiro/src/exercicio16e17e18e19/Aluno.java

```diff
@@ -0,0 +1,30 @@
+package exercicio16e17e18e19;
+
+public class Aluno {
+
+	private String nome;
+	private String rg;
+	private int idade;
+	
+	public Aluno(String nome, String rg, int idade ) {
+		this.nome = nome;
+		this.rg = rg;
+		this.idade = idade;
+	}
+	
+	public boolean equals(Object obj) {
```

Comment:
Pelo eclipse é possível gerar hashCode e equals, com maior facilidade. Utilize o atalho ctrl +3, que vai abrir uma aba, e nela você consegue escolher o que deseja gerar, se é getter and setter, hashCode and equals e outros.